slider.html

```diff
@@ -0,0 +1,67 @@
+<!DOCTYPE html>
+<html>
+<head>
+	<meta charset="utf-8">
+	<meta http-equiv="X-UA-Compatible" content="IE=edge">
+	<title>Slider</title>
+	<link rel="stylesheet" href="css/style.css">
+	<script type="text/javascript" src="js/slider.js"></script>
+
+</head>
+<body>
+	<section>
+	<div class="container">
+		<div id="slider">
+			<div class="item" style="background: url('img/imagem1.jpg');">
+				<div style="padding-top: 500px; width: 100%; text-align: center; color: #fff; "> 
+					<h1>AQUI INICIAMOS GRANDES COISAS</h1>
+				</div>
+			</div>
+			<div class="item" style="background: url('img/imagem2.jpg');"></div>
+			<div class="item" style="background: url('img/imagem3.jpg');"></div>
+			<button class="button display-left" onclick="plusDivs(-1)">&#10094;</button>
+			<button class="button display-right" onclick="plusDivs(+1)">&#10095;</button>
+		</div>	
+	</div>
+	</section>
+
+	<script>
+	var slideIndex = 1;
+		showDivs(slideIndex);
+
+	function plusDivs(n) {
+		showDivs(slideIndex += n);
+	}
+
+	function showDivs(n) {
```

Comment:
vc que criou essas funcoes?

Reply:
claro que não kkkkk

Reply:
kkk entao vamos apagar. e vc vai fazer do zero. E to falando serio 😈

Reply:
concodo